.prettierignore

```diff
@@ -1,9 +1,9 @@
 # Ignore artifacts:
-build
+docs/build
 coverage
-node_modules
-cypress
-src/docs/shared/types/contentful.ts
-schema.graphql
+**/node_modules
```

Comment:
some differences in / vs ** in this and gitignore. intentional?

Reply:
** is to ignore all the node_modules folders in the monorepo

Reply:
i meant that we have /docs/build vs docs/build and /**/node vs **/node